apps/core/mutations.py

```diff
@@ -35,6 +37,16 @@
 )
 
 
+class ColumnType(graphene.ObjectType):
+    key = graphene.String()
+    label = graphene.String()
```

Comment:
We can use `KeyLabelType` here as it is already defined.

Reply:
Also, make the type strict here.

Reply:
Done